demos/demo-context-dependencies/app/Main.hs

```diff
@@ -19,7 +19,7 @@ database = Label :: Label "database" DatabaseContext
 
 introduceDatabase = introduceWith "Introduce database" database $ \action ->
   bracket (debug "Spinning up DB..." >> return MySQLDatabaseContext)
-          (\db -> debug "Tearing down DB..." >> return ())
+          (\db -> void (debug "Tearing down DB..."))
```

Comment:
Please keep these returns, I think hlint is being overly fussy here and the returns are more readable for these demos.